packages/models-library/src/models_library/projects.py

```diff
@@ -66,17 +66,21 @@ class ProjectCommons(BaseModel):
 
     @validator("thumbnail", always=True, pre=True)
     @classmethod
-    def convert_empty_str_to_none(v):
+    def convert_empty_str_to_none(cls, v):
```

Comment:
or a `@staticmethod` if you don't need `cls`?

Reply:
true